app/views/trainees/_filters.html.erb

```diff
@@ -112,13 +112,14 @@
         <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
           <%= t("views.trainees.index.filters.status") %>
         </legend>
-        <div class="govuk-checkboxes govuk-checkboxes--small">
-          <% TraineeFilter::STATES.each do |state, _| %>
-            <% next if state == "draft" %>
-            <div class="govuk-checkboxes__item">
-              <%= check_box_tag "state[]", state, checked?(filters, :state, state), id: "state-#{state}", class: "govuk-checkboxes__input" %>
-              <%= label_tag "state-#{state}", label_for("state", state), class: "govuk-label govuk-checkboxes__label" %>
-            </div>
+        <v class="govuk-checkboxes govuk-checkboxes--small">
+          <% TraineeFilter::STATUSES.each do |status, _| %>
+            <% unless status == "course_not_yet_started" && any_not_yet_started_trainees? %>
```

Comment:
i found this unless a little hard to read but could just be me. Is it worth implementing a next if instead?

```
next if status == "course_not_yet_started" && !trainees_not_started_yet?
```

not sure which is easier to read

Reply:
or maybe you could invert the check

```
next if status == "course_not_yet_started" && all_trainees_started?
```

Reply:
i like the `all_trainees_started` check, reads nicely 👍